packages/clay-layout/src/ContainerFluid.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+
+import Container from './Container';
+
+interface IProps
+	extends Omit<
+		React.ComponentProps<typeof Container>,
+		'fluid' | 'fluidSize'
+	> {
+	size?: React.ComponentProps<typeof Container>['fluidSize'];
+}
+
+const ClayContainerFluid: React.FunctionComponent<IProps> = ({
```

Comment:
LGTM!

nit: Should it `FluidContainer` instead of `ContainerFluid`?

Reply:
Since its `.container-fluid`, it made more sense to me in keeping it consistent with that. I also prefer `NounDescriber`(if that makes sense)

Reply:
I think it's weird... because in the other components we did the opposite?

I'm thinking now at `ContentSection`, `ContentRow` and `ContentCol`...  it might just be my lack of familiarity with the grammar, but those feel like `DescriberNoun`. Basically i can tell because describer is common and noun is the thing that changes at the end.

It's true that we don't have many such cases in Clay, but others I found that also seem to contradict this are `ItemField`, `ItemText` and `ItemTitle`...

Reply:
Ah, yeah, never mind... I think I see it now 😂 👍

Reply:
Ah, yeah, never mind, I think I see it now... 😂 👍

Reply:
Or maybe not! 😂 @wincent, @pat270, some native english speakers here, please? I'm fine either way 👍

Reply:
Just to state my confusion.

`ContentSection`, `ContentRow` and `ContentTitle`... one would assume as follows that we might have a `ContentContainer`.

`ItemField`, `ItemText`, `ItemTitle`... similarly, I'd think an `ItemContainer` seems logical...

From the above, `FluidContainer` seems to follow the sequence... but I was never good at this 😂

Reply:
Native English speaker here... English laughs at your pathetic request to have any kind of consistent rules.

![fart-in-your-general-direction](https://user-images.githubusercontent.com/7074/81941827-9a84a100-95f9-11ea-8b36-52e9bbd26575.gif)

(I know the gif is "French", but it accurately summarizes the English attitude to consistency.)

The general grammatical pattern in English is to put modifiers (adjectives) before nouns (eg. "red car"). Things get complicated because English is quite happy to use nouns as modifers too (ie. noun like "money" being used like an adjective in the term "money bag") — but still, the modifier goes first. Unlike in Spanish, which is pretty much the opposite (eg. "coche rojo").

But programming is another layer above English, so there are other concerns beyond mere grammar. For example, we like to group related things together using a prefix. That's why `ItemField`, `ItemText`, `ItemTitle` start with a common prefix: the "thing" here is "Field", "Text", "Title", but the common relationship is that they are all the "thing" of an "Item", so we extract the common bit out and stick it at the front. And even more obviously, everything in Clay starts with "Clay", not because "Clay" is a modifier or an adjective or a noun or anything — it's the common root that it all relates to.

With respect to `ContainerFluid` vs `FluidContainer`, I have no idea. The "thing" it is is a container, and "Fluid" is the modifier that describes the container, so I'd expect it to be `FluidContainer`. But as @bryceosterhaus  said, he was being consistent with `.container-fluid`.

Reply:
yeah the more I think about it I really have no idea why somethings sounds "more right" than the other way around. So I think I'll just rest my decision based on the class name itself.